.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,145 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: 
+      frequencies:
+      - hourly:
+          hour: 3
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: doc-bug
+      - isNotLabeledWith:
+          label: okr-health
+      actions:
+      - addLabel:
+          label: okr-health
+    eventResponderTasks:
+    - if:
+      - payloadType: Pull_Request
+      then:
+      - inPrLabel:
+          label: in-pr
+      description: Add in-pr label to issues
+    - if:
+      - payloadType: Pull_Request
+      - isAction:
+          action: Opened
+      - and:
+        - not:
+            activitySenderHasPermission:
+              permission: Admin
+        - not:
+            activitySenderHasPermission:
+              permission: Admin
+        - not:
+            activitySenderHasPermission:
+              permission: Write

```

Comment:
It looks like "maintain" was converted to "admin". We need to differentiate between those two roles as the previous configuration did. @JohannesLampel, is this a bug in the generation?

Previous configuration: https://github.com/dotnet/docs/blob/0135bf9df107eb22edcf0f0096aea87a0399712c/.github/fabricbot.json#L97-L129

Reply:
Yes, at least kind of. The maintain permission does not exist anymore. However, I will update the tool to assign Write instead. This is in line with GitHub and should be the same experience that was there before.

Reply:
I pushed a change to resolve this directly in this PR.